src/component/Header/index.jsx

```diff
@@ -15,9 +39,11 @@ const Header = () => (
     <Navbar.Collapse>
       <Nav className="ml-auto">
         <Image className={styles['setting-logo']} src={settingLogo} rounded />
-        <Nav.Link href="#setting">Setting</Nav.Link>
-        <Image className={styles['login-logo']} src={loginLogo} rounded />
-        <Nav.Link href="#login">Login</Nav.Link>
+        <Nav.Link href="#setting">
+          <SettingModal />
+        </Nav.Link>
+        <Image className={`${styles['login-logo']}`} src={loginLogo} rounded />
+        <Nav.Link href="#login" className="text-white">Login</Nav.Link>
```

Comment:
Can you remove this too because now it looks really ugly next to the Settings button. Since we have real users of the application now, we don't want to deploy code that makes the UI look ugly.